src/Settings/components/transliteration.jsx

```diff
@@ -0,0 +1,92 @@
+import React from "react";
+import { useState } from "react";
+import { Text } from "react-native";
+import { ListItem, BottomSheet, Avatar, Icon, Switch } from "@rneui/themed";
+import STRINGS from "../../common/localization";
+import {
+  setTransliteration,
+  toggleTransliteration,
+  TRANSLITERATION_LANGUAGES,
+} from "../../common/actions";
+import { useSelector } from "react-redux";
+import colors from "../../common/colors";
+import styles from "../styles";
+
+function TransliterationComponent({ isNightMode, dispatch }) {
```

Comment:
Why are we passing dispatch as an argument in each component and not using useDispatch hook function?

Reply:
I have called the usDispatch once and used it's instance to all other children component. so that we don't have to call useDispatch again and again.